cadCAD/engine/execution.py

```diff
@@ -44,6 +47,46 @@ def single_proc_exec(
         results.append(flatten(result))
     return flatten(results)
 
+
+def process_executor(params):
```

Comment:
`process_executor` is being defined twice. Remove or refactor the one that's not being used.